Makefile

```diff
@@ -35,18 +35,20 @@ endif
 .PHONY: all
 all: clean test build
 
-$(CMDS):
+$(CMDS): vendor
```

Comment:
Understanding that we're just adding a single new target, it almost seems like we should recognize the "all targets" use-case here and add a 
```make
.PHONY: environment
environment: vendor
```
and add the dependency `environment` to every target instead.  This would allow us to scale out if we discover future "for all targets" kind of things.

Reply:
This could be coming from my lack of familiarity with the project but for the `vendor` target at least this may not be necessary, since not all targets actually require that the vendor directory be present. That may add some unnecessary churn to some of the make targets here and there.